docs/schema/reference.md

```diff
@@ -655,7 +655,11 @@ For delivery milestones, if there is a time frame for delivery, use `.dueAfterDa
 
 ### Value
 
-Financial values should be published with a currency attached. 
+Financial values should be published with a currency attached. There are three fields available for providing the value amount:
+
+* When it is known that the value includes all taxes use `value/amountGross`;
+* When it is known that the value excludes all taxes use `value/amountNet`;
+* When it is unknown if the value includes all taxes or not use `value/amount`.
```

Comment:
The original change doesn't cover all the scenarios discussed in #817. Most of the text was just repeating the semantics of the fields, which doesn't demand repetition.

````suggestion
`currency`, `amountNet` and `amountGross` should be provided, wherever possible.

`amount` is defined as:

```{field-description} ../../build/current_lang/release-schema.json /definitions/Value/properties/amount
```

If both the `amountNet` and the `amountGross` match this definition, enter the `amountNet` as the `amount`.
````